.github/workflows/release.yaml

```diff
@@ -0,0 +1,27 @@
+name: Release
+on:
+  workflow_dispatch:
+jobs:
+  make_github_release:
+    uses: datajoint/.github/.github/workflows/make_github_release.yaml@main
+  pypi_release:
+    needs: make_github_release
+    uses: datajoint/.github/.github/workflows/pypi_release.yaml@main
+    secrets:
+      TWINE_USERNAME: ${{secrets.TWINE_USERNAME}}
+      TWINE_PASSWORD: ${{secrets.TWINE_PASSWORD}}
+    with:
+      UPLOAD_URL: ${{needs.make_github_release.outputs.release_upload_url}}
```

Comment:
```suggestion
```

Reply:
This step will fail during the release with the new `setup.py` so we can remove it.